airflow-core/src/airflow/ui/src/theme.ts

```diff
@@ -34,140 +36,350 @@ const customConfig = defineConfig({
-          "600": { value: "#808080" },
-          "700": { value: "#616161" },
-          "800": { value: "#404040" },
-          "900": { value: "#212121" },
-          "950": { value: "#0F0F0F" },
-        },
-        skipped: defaultConfig.theme?.tokens?.colors?.pink ?? {},
-        up_for_reschedule: defaultConfig.theme?.tokens?.colors?.cyan ?? {},
-        up_for_retry: defaultConfig.theme?.tokens?.colors?.yellow ?? {},
-        upstream_failed: defaultConfig.theme?.tokens?.colors?.orange ?? {},
-        // lime
-        running: {
-          "50": { value: "#EFFBEF" },
-          "100": { value: "#DEF7DE" },
-          "200": { value: "#B9EEB9" },
-          "300": { value: "#98E698" },
-          "400": { value: "#78DE78" },
-          "500": { value: "#53D553" },
-          "600": { value: "#32CD32" },
-          "700": { value: "#269C26" },
-          "800": { value: "#196719" },
-          "900": { value: "#0D350D" },
-          "950": { value: "#061906" },
-        },
-        // violet
-        restarting: {
-          "50": { value: "#F6EBFF" },
-          "100": { value: "#EDD6FF" },
-          "200": { value: "#D9A8FF" },
-          "300": { value: "#C880FF" },
-          "400": { value: "#B657FF" },
-          "500": { value: "#A229FF" },
-          "600": { value: "#8F00FF" },
-          "700": { value: "#6E00C2" },
-          "800": { value: "#480080" },
-          "900": { value: "#260042" },
-          "950": { value: "#11001F" },
-        },
-        // mediumpurple
-        deferred: {
-          "50": { value: "#F6F3FC" },
-          "100": { value: "#EDE7F9" },
-          "200": { value: "#DACEF3" },
-          "300": { value: "#C8B6ED" },
-          "400": { value: "#B9A1E7" },
-          "500": { value: "#A689E1" },
-          "600": { value: "#9370DB" },
-          "700": { value: "#6432C8" },
-          "800": { value: "#412182" },
-          "900": { value: "#211041" },
-          "950": { value: "#100821" },
-        },
-        // tan
-        scheduled: {
-          "50": { value: "#FBF8F4" },
-          "100": { value: "#F8F3ED" },
-          "200": { value: "#F1E7DA" },
-          "300": { value: "#E8D9C4" },
-          "400": { value: "#E1CDB2" },
-          "500": { value: "#DAC1A0" },
-          "600": { value: "#D2B48C" },
-          "700": { value: "#B9894B" },
-          "800": { value: "#7D5C31" },
-          "900": { value: "#3E2E18" },
-          "950": { value: "#21180D" },
-        },
-        // lightblue
-        none: {
-          "50": { value: "#F7FBFD" },
-          "100": { value: "#F3F9FB" },
-          "200": { value: "#E4F2F7" },
-          "300": { value: "#D8ECF3" },
-          "400": { value: "#C8E5EE" },
-          "500": { value: "#BDDFEB" },
-          "600": { value: "#ADD8E6" },
-          "700": { value: "#5FB2CE" },
-          "800": { value: "#30819C" },
-          "900": { value: "#18414E" },
-          "950": { value: "#0C2027" },
-        },
```

Comment:
We are missing the `none` color palette. How are `no status` tasks displayed ?

Reply:
The `none` color palette is defined in the semantic tokens section and uses the "gray" color palette.